pkg/kubelet/kubelet.go

```diff
@@ -1713,6 +1713,9 @@ func (kl *Kubelet) syncPod(ctx context.Context, updateType kubetypes.SyncPodType
 	// Fetch the pull secrets for the pod
 	pullSecrets := kl.getPullSecretsForPod(pod)
 
+	// Ensure the pod is being probed
+	kl.probeManager.AddPod(pod)
```

Comment:
https://github.com/kubernetes/kubernetes/blob/c6970e64528ba78b74bf77b86f9b78b7b61bd0cd/pkg/kubelet/prober/prober_manager.go#L188
the `syncPod ` is reentrant, so shall we set the log level above more friendly?

Reply:
This patch does not change the logic within the AddPod function.

Reply:
Good point. I bumped the log level to 4 for these.